src/main.tsx

```diff
@@ -1,10 +1,92 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.tsx";
-// import './index.css'
+import { ThemeProvider, createTheme } from "@mui/material";
+
+import "@fontsource/chakra-petch";
+
+const theme = createTheme({
+  palette: {
+    background: {
+      paper: "#fff",
+    },
+    text: {
+      primary: "#000",
+      secondary: "#46505A",
+    },
+    action: {
+      active: "#000",
+    },
+  },
+  typography: {
+    h2: {
+      fontSize: "2rem",
+      fontWeight: 600,
+      fontFamily: "Chakra Petch, sans-serif",
+    },
+    h3: {
+      fontSize: "1.5rem",
+      fontWeight: 600,
+      fontFamily: "Chakra Petch, sans-serif",
+    },
+    h4: {
+      fontSize: "1.2rem",
+      fontWeight: 600,
+      fontFamily: "Chakra Petch, sans-serif",
+    },
+  },
+  components: {
+    MuiTextField: {
+      styleOverrides: {
+        root: {
+          display: "block",
+          width: "100%",
+          "& .MuiInputBase-root": {
+            width: 350,
+            maxWidth: "100%",
+            display: "block",
+            marginBottom: 10,
+          },
+          "& .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline": {
+            borderColor: "#000",
+          },
+        },
```

Comment:
This is a bit dirty hack that dont really work, number textfiled looks broken. Idea of theme config is to specify most simple and generic components styling. For example we would like to make ALL of our buttons rounded or with some kind of hover... But here looks like you'r using it for make textfields with width: "100%", and this style applys to all of textfileds in the project. This is bad because imagine if in the future one of them should be 50% and so on... 
Solution here is to write specific styles that will be different all throught the project (as layout for example) for every single componet. In this case just drop misused styles in config and use defult prop `fullwidth` in every textField

<img width="398" alt="Screenshot 2024-04-28 at 14 27 44" src="https://github.com/proga-react-practice/xi-sofiia-stanishevska/assets/103277184/b0a5ab4f-3085-42df-a2df-e5d6214ac88a">